### DISCUSSION TITLE
Fixed ip e DHCP in base a SSID su Tasmota

### DISCUSSION THREAD

**Discussion by XandroTs** [indomus/forum#2635] [Networking]
Salve, vorrei replicare quello che faccio su Shelly anche su Tasmota. Su Shelly posso configurare i due SSID ognuno con impostazioni di rete diverse e quindi il primo lo metto in IP fisso e il secondo in DHCP. Su Tasmota posso mettere 2 SSID ma le impostazioni di rete sono comuni ai 2 SSID.
In sostanza vorrei configurare su Tasmota 2 SSID uno in IP fisso e uno in DHCP, secondo voi è possibile?
Grazie a tutti

**Comment by snakuzzo** [comment]
Credo non si possa fare. Puoi sì impostare i due SSID diversi, ma la configurazione manuale dell'IP credo sia unica.
Domanda... non puoi utilizzare l'assegnazione statica in base al MAC address direttamente sul router ? Alla fine il risultato, per quello che ti serve, sarebbe lo stesso

**Comment by XandroTs** [comment]
Preferisco che sia il dispositivo ad avere l'ip impostato e comunque per pigrizia non ho mai creato un DHCP sulla rete IOT, speravo ci fosse un metodo a me sconosciuto.
Quando collego i dispositivi tasmotati al cellulare in tethring simulando la rete IOT non si allineano perche i nuovi android non hanno piu la subnet impostabile la danno loro a caso.
